base/uk.ac.stfc.isis.ibex.scriptgenerator/defined_actions/action_loader.py

```diff
@@ -44,7 +45,7 @@ def getParameters(self) -> list:
 
     def doAction(self, list_of_arguments) -> None:
         """Executes the action with the parameters provided"""
-        self.action.run(*list_of_arguments)
+        self.action.run(**list_of_arguments)
```

Comment:
Currently they are all passed as keyword arguments so yes it must be documented. However I think we should leave cast_parameters_to as able to accept args because the API may change if we decide we want to handle args later or if there is a use case for it elsewhere.

Reply:
See documentation at https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Script-generator-high-level-design and in code for ActionDefinition base class.